readthedocs/profiles/views.py

```diff
@@ -399,6 +399,7 @@ def post(self, request, *args, **kwargs):
         else:
             projects = get_valid_projects_missing_migration(request.user)
 
+        count = 0
```

Comment:
I would use a more descriptive name here: `n_projects_migrated` for example.

```suggestion
        n_projects_migrated = 0
```